ontology.md

```diff
@@ -165,16 +165,16 @@ Data is encoded using the [SOSA](https://www.w3.org/TR/vocab-ssn/) ontology. Loo
 1. `sosa:Observation`: Contains numeric or categorical data about an observation which is stored as literal values, accessible through `sosa:hasSimpleResult`.
 2. `sosa:ObservationCollection`: Nodes of this type point to one or more `sosa:Observation` nodes through the relation `sosa:hasMember`.
 
-Rather than using the classnames above, KnowWhereGraph subclasses them into more specific instances. For example, thunderstorm events are connected to `kwg-ont:ImpactObservationCollection`, which is a subclass of `sosa:ObservationCollection`. **Nodes can be connected to multiple observation collections, of different types**
+Rather than using the class names above, KnowWhereGraph subclasses them into more specific instances. For example, thunderstorm events are connected to `kwg-ont:ImpactObservationCollection`, which is a subclass of `sosa:ObservationCollection`. **Nodes can be connected to multiple observation collections, of different types.**
 
-This won't make a huge impact on the way you query data, but allows you to be more specific in the kinds of data that you want.
+This won't make a huge impact on the way you query data, but it allows you to be more specific in the kinds of data that you want.
 
 When reading and using the SOSA ontology, it's important to buy into the concept of properties and observations:
 
 1. Properties: Things being observed. This could be the air temperature, number of deaths, obesity rates, number of beds that a hospital has, etc.
 2. Observation: The act that resulted in a property being observed. It provides the context *around* the property, such as the value of the property or the name of the observation. This would be the literal *number* of hospital beds.
```

Comment:
I don't understand the explanation and example for `2. Observation`. If an Observation is an act, it should be a verb, like "counting". The example given, "the literal *number* of hospital beds", in my head is a value like "56". This isn't a verb. I'm afraid I don't understand SOSA well enough to know whether the explanation needs adjusting or the example given needs changing (or if I'm misunderstanding this for another reason).

Reply:
I think I'm trying to say that an observation `sosa:hasSimpleResult`. So the act of observing produces a simple result - which has the number. There's a high chance that I don't understand SOSA enough to make this philosophical claim though!